src/main/java/com/technocredits/orangeHRMAutomation/util/PropertyFileReader.java

```diff
@@ -0,0 +1,23 @@
+package com.technocredits.orangeHRMAutomation.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyFileReader {
+	//String loginPropertiesPath = "E:\\Aditya\\Projects\\Oct20\\TechnoCreditsOrangeHRMGroup1\\src\\main\\java\\com\\technocredits\\orangeHRMAutomation\\config\\LoginPageProperties.properties";
```

Comment:
Remove this type of dead code,

Reply:
Code is removed now. Also, pushed the new code for the AddEmployee.